packages/service-library/src/web-api/web-api-error-codes.ts

```diff
@@ -121,6 +122,15 @@ export class WebApiErrorCodes {
         },
     };
 
+    public static malformedRequest: WebApiErrorCode = {
+        statusCode: 400,
+        error: {
+            code: 'MalformedBody',
+            codeId: 4011,
+            message: 'Please validate the request body against the schema for this version and endpoint.',
```

Comment:
May be rephrase to something like: The request body does not match to the API schema for the given API version.

Reply:
Done.